common-tools/clas-tracking/pom.xml

```diff
@@ -16,8 +16,8 @@
   <dependencies>
     <dependency>
         <groupId>org.ejml</groupId>
-        <artifactId>ejml-all</artifactId>
-        <version>0.44.0</version>
+        <artifactId>ejml-simple</artifactId>
+        <version>0.40</version>
```

Comment:
any reason not to use `ejml-simple` version 0.44? Or should we tell Dependabot to keep this at 0.40?

Reply:
I doubt we know what ejml-simple is ...